Userland/Libraries/LibCore/System.cpp

```diff
@@ -245,6 +245,27 @@ ErrorOr<void> munmap(void* address, size_t size)
     return {};
 }
 
+ErrorOr<int> anon_create(size_t size, [[maybe_unused]] int options)
+{
+    int fd = -1;
+#if defined(__serenity__)
+    fd = ::anon_create(size, options);
+    if (fd < 0)
+        return Error::from_errno(errno);
```

Comment:
```suggestion
```
This is duplicate with lines 264-265.
(note the lines 264-265 are more usefull as they cover the case where this is built for neither `serenity` or `linux`)

Reply:
Interesting, the original code I copied this from has that as well.